x/cronos/types/proposal.go

```diff
@@ -0,0 +1,62 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	// ProposalTypeChange defines the type for a TokenMappingChangeProposal
+	ProposalTypeChange = "TokenMappingChange"
```

Comment:
const name can be less generic, in case we add more type in the future

such as `TokenMappingChangeType`

Reply:
renamed to `ProposalTypeTokenMappingChange`.